nixos/modules/services/accessibility/speechd.nix

```
@@ -8,25 +8,86 @@ let
   cfg = config.services.speechd;
   inherit (lib)
     getExe
+    mapAttrs'
     mkEnableOption
     mkIf
     mkPackageOption
+    mkOption
     ;
 in
 {
   options.services.speechd = {
     # FIXME: figure out how to deprecate this EXTREMELY CAREFULLY
     # default guessed conservatively in ../misc/graphical-desktop.nix
     enable = mkEnableOption "speech-dispatcher speech synthesizer daemon";
+
     package = mkPackageOption pkgs "speechd" { };
+
+    settings = mkOption {
```

```suggestion
    config = mkOption {
```

Settings is usually in RFC42 format